modules/custom/moj_resources/src/SuggestedContentApiClass.php

```diff
@@ -129,88 +128,79 @@ private function getSuggestions($nid, $number, $prison)
    * @return array
    */
   private function getTagIds($tags) {
-    $tag_ids = [];
-    $number_tags = count($tags);
+    $tagIds = [];
+    $numberTags = count($tags);
 
-    for ($i = 0; $i < $number_tags; $i++) {
-      array_push($tag_ids, $tags[$i]->target_id);
+    for ($i = 0; $i < $numberTags; $i++) {
```

Comment:
Could this be an `array_map()`?

Reply:
@LucasCairns probably, will have a nosey!